packetbeat/docs/faq-mysql-ssl.asciidoc

```diff
@@ -0,0 +1,27 @@
+[float]
+[[mysql-no-data]]
+=== {beatname_uc} isn't capturing MySQL performance data?
+
+You may be listening on the wrong interface or trying to capture data sent over
+an encrypted connection. {beatname_uc} can only monitor MySQL traffic if it is
+unencrypted. To resolve your issue:
+
+* Make sure {beatname_uc} is configured to listen on the `lo0` interface:
++
+[source,shell]
+-----
+packetbeat.interfaces.device: lo0
+-----
+
+* Make sure the client programs you are monitoring run `mysql` with SSL
+disabled. For example:
++
+[source,shell]
+-----
+mysql --protocol tcp --host=127.0.0.1 --port=3306 --ssl-mode=DISABLED
+-----
+
+IMPORTANT: When SSL is disabled, the connection between the MySQL client and
+server is unencrypted, which means that anyone with access to your network may
+be able to inspect data sent between the client and server. If MySQL is running
+in an untrusted network, it's not advisable to disable encryption.
```

Comment:
Or if we ant to be even more direct, we can say:

If MySQL is running in an untrusted network, disabling encryption poses a security risk.